Dashboard/app/js/lib/views/views.js

```diff
@@ -88,7 +88,8 @@ Ember.Handlebars.registerHelper('tooltip', function (i18nKey) {
 
 
 Ember.Handlebars.registerHelper('placemarkDetail', function () {
-  var answer, markup, question, cascadeJson, optionJson, cascadeString = "", questionType, imageSrcAttr, signatureJson;
+  var answer, markup, question, cascadeJson, optionJson, cascadeString = "",
+  questionType, imageSrcAttr, signatureJson, phtoJson;
```

Comment:
The declared `phtoJson` var has a different name from the actually used one it seems?